src/ensembl/src/shared/accordion/components/Accordion.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import DisplayName from '../helpers/DisplayName';
+import { DivAttributes } from '../helpers/types';
+import { Consumer, Provider } from './AccordionContext';
+import { UUID } from './ItemContext';
+import styles from '../css/Accordion.scss';
+
+type AccordionProps = Pick<
+  DivAttributes,
+  Exclude<keyof DivAttributes, 'onChange'>
+> & {
+  preExpanded?: UUID[];
+  allowMultipleExpanded?: boolean;
+  allowZeroExpanded?: boolean;
```

Comment:
If allowMultipleExpanded and allowZeroExpanded are among default props, then they are always present, which means there is no need in the question mark.

Reply:
That's done